<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemSettings</class>
 <widget class="QWidget" name="ItemSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>203</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="param_ed" name="item_border_thick">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize1" stdset="0">
    <size>
     <width>50</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="labelBGColor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string> BG color</string>
   </property>
  </widget>
  <widget class="ColorLabel" name="BGcolor">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelBorderColor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>61</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Border color</string>
   </property>
  </widget>
  <widget class="ColorLabel" name="BorderColor">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelBorderThick">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Border thick</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWidth">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>  Width</string>
   </property>
  </widget>
  <widget class="param_ed" name="item_width">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize1" stdset="0">
    <size>
     <width>50</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="labelHight">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Height</string>
   </property>
  </widget>
  <widget class="param_ed" name="item_height">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize1" stdset="0">
    <size>
     <width>50</width>
     <height>20</height>
    </size>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorLabel</class>
   <extends>QLabel</extends>
   <header>colorlabel.h</header>
  </customwidget>
  <customwidget>
   <class>param_ed</class>
   <extends>QLineEdit</extends>
   <header location="global">param_ed.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
